app/styles/components/_footer.scss

```diff
@@ -7,6 +7,8 @@ $svg-height-mobile: 16px;
 
 $icon-size: 25px;
 
+$footer-break-navlinks: 940px;
```

Comment:
I have the feeling that this could be better named, since it is referring to the specific behaviour of the footer links. Something like `$footer-mobile-break`, which would allow for the same value to be used in other media queries.

It's not a big deal and you can leave it like that since it's applied to a really specific case. Just my two cents.